view/frontend/templates/subscribe.phtml

```diff
@@ -0,0 +1,33 @@
+<?php
+// @codingStandardsIgnoreFile
+
+/** @var \Magento\Newsletter\Block\Subscribe $block */
+
+?>
+<div class="block newsletter">
+    <div class="title"><strong><?= $block->escapeHtml(__('Newsletter')) ?></strong></div>
+    <div class="content">
+        <form class="form subscribe"
+            novalidate
+            action="<?= $block->escapeUrl($block->getFormActionUrl()) ?>"
+            method="post"
+            data-mage-init='{"validation": {"errorClass": "mage-error"}}'
+            id="newsletter-validate-detail">
+			<?php /* @noEscape */ echo $block->getBlockHtml('formkey'); ?>
+            <div class="field newsletter">
+                <label class="label" for="newsletter"><span><?= $block->escapeHtml(__('Sign Up for Our Newsletter:')) ?></span></label>
+                <div class="control">
+                    <input name="email" type="email" id="newsletter"
+                                placeholder="<?= $block->escapeHtmlAttr(__('Enter your email address')) ?>"
+                                data-validate="{required:true, 'validate-email':true}"/>
+                    <input type="hidden" name="hpt-url" autocomplete="off" />
```

Comment:
In Rule's honey pot implementation we hide the element with css, not directly in the html. So, I'm not sure how this would work. Have you done it this way before, does it work?

An example of our implementation: 
https://docs.rule.se/article/283-rule-botstop-spam-stop-via-api